### ISSUE TITLE
Pytorch --> ONNX --> Tensorflow. How to translate the Bounding Box Voting part?

### ISSUE THREAD

**Issue by MroutsideturnAF** [biubug6/Pytorch_Retinaface#164]
When transforming the model to Onnx and then to Tensorflow. It returns output in the shape of 3 Tensors: 

` loaded model outputs  = [<tf.Tensor 'Identity:0' shape=(1, 16800, 4) dtype=float32>, <tf.Tensor 'Identity_1:0' shape=(1, 16800, 2) dtype=float32>, <tf.Tensor 'Identity_2:0' shape=(1, 16800, 10) dtype=float32>]`

This format corresponds to `loc, conf, landms = net(img)  # forward pass` from detect.py

Is there a recommendable way to translate the bounding box part to Tensorflow aswell?

**Comment by alicera** [comment]
Hi AndyMcPython ,
May I ask your network  learning for resnet50 and mobilenet?

**Comment by MroutsideturnAF** [comment]
Hi @alicera, 
I use the resnet50.

**Comment by alicera** [comment]
How to adjust the learning rate for better training？

**Comment by lihaoxiang1989** [comment]
Do you mean the NMS operation? You may be interested in this https://github.com/lihaoxiang1989/Tensorrt-CV/tree/master/Retinaface
